supertokens_python/recipe/thirdpartyemailpassword/recipe.py

```diff
@@ -96,7 +100,7 @@ def __init__(self, recipe_id: str, app_info: AppInfo,
             self.email_verification_recipe = email_verification_recipe
         else:
             self.email_verification_recipe = EmailVerificationRecipe(recipe_id, app_info,
-                                                                     self.config.email_verification_feature)
+                                                                     self.config.email_verification_feature, EmailVerificationIngredients(None))
```

Comment:
Do not pass none to this

Reply:
It's fine for now. It's passed in the relevant PR for the tpep recipe https://github.com/supertokens/supertokens-python/pull/163/files#diff-1d423b5d85fc2ebb83925be074e6758f869042c73d9f9a8ac33b96d38757b71eR132